lib/prepareReturnValue.cjs

```diff
@@ -42,11 +42,26 @@ function prepareReturnValue(stylelintResults, maxWarnings, formatter, cwd) {
 		cwd,
 		errored,
 		results: [],
-		output: '',
+		report: '',
+
+		// TODO: Deprecated. Remove in the next major version.
+		get output() {
+			if (!this._outputWarned) {
+				console.warn('`output` is deprecated. Use `report` or `code` instead.');
+				this._outputWarned = true;
```

Comment:
question

We only have 4 more files that use that pattern:
https://github.com/stylelint/stylelint/blob/main/lib/utils/FileCache.js#L35
https://github.com/stylelint/stylelint/blob/main/lib/rules/max-line-length/__tests__/index.js#L6
https://github.com/stylelint/stylelint/blob/main/lib/utils/isStandardSyntaxTypeSelector.js#L25
https://github.com/stylelint/stylelint/blob/main/lib/utils/whitespaceChecker.js#L212

Should we review if it's consistent?

Reply:
We have used the `_` prefix for internal (private) properties, for example:

https://github.com/stylelint/stylelint/blob/3d79ac7ea5202af65bf990878656722bfd71f13a/types/stylelint/index.d.ts#L352-L357

Reply:
There's a difference between `@internal` and [`@private`](https://jsdoc.app/tags-private.html).
TBD in a refactor PR i.e. not this one